README.md

```diff
@@ -1,38 +1,19 @@
 # ChatEngine Server
 
-The application that adds a keyset and the PubNub Function Module to a user's account automatically on behalf of ChatEngine.
+This repository contains the REST API running as a PubNub function that is required for
+the [ChatEngine Framework](https://github.com/pubnub/chat-engine/) to operate.
 
-The folders `/app` and `/lib` are generated by the build process, don't edit their contents.
+## Setup
 
-To update ChatEngine Server, edit the contents within ```src/functions/*``` and then rebuild. Server will be output to ```app/functions/server.js```.
+First, configure your application in the PubNub administration UI.
 
-# Gulp
-```
-$ gulp compile // Default task - Does clean, build, test
-$ gulp clean   // Deletes the last build (./app contents)
-$ gulp build   // Compiles full stack and moves all build files to /app
-$ gulp test    // Runs unit and integration tests in /test
-```
+[How to Setup your account for ChatEngine](guides/ChatEngine_Application_and_Key_Provisioning.md)
```

Comment:
"Refer to [How to set up your account for ChatEngine](..) for more details."?

(Also "set up" is two words in this situation)